samples/src/source/tasks/ContentTask.bs

```diff
@@ -0,0 +1,29 @@
+import "pkg:/source/roku_modules/log/LogMixin.brs"
+import "pkg:/source/roku_modules/maestro/core/Utils.brs"
+import "pkg:/source/roku_modules/maestro/core/Request.brs"
+import "pkg:/source/roku_modules/maestro/core/MIOC.brs"
+
+@task("ContentTask", "Task")
+@strict
+class ContentTask
+  private log = new log.Logger("ContentTask")
+
+  @inject("styleManager")
+  private styleManager
+
+  ' @injectClass("mc.net.Request")
```

Comment:
why is this commented out? was there a problem with this? - I think it's more correct to instnatiate it, as it goes; but wondering - if this code is not used, remove it.